go.mod

```
@@ -1,6 +1,6 @@
 module github.com/argoproj/argo-workflows/v3
 
-go 1.22
+go 1.22.0
```

```suggestion
go 1.22.5
```

1.22.5 looks like it's latest: https://github.com/golang/go/releases/tag/go1.22.5

Does that work?

---

updated